eva/storage/opencv_storage_engine.py

```diff
@@ -57,7 +60,18 @@ def drop(self, table: DataFrameMetadata):
             logger.exception(f"Failed to drop the video table {e}")
 
     def write(self, table: DataFrameMetadata, rows: Batch):
-        pass
+        df = rows.frames
+        try:
+            dir_path = Path(table.file_url)
+            for video_file_path in df["video_file_path"]:
+                video_file = Path(video_file_path)
+                shutil.copy2(str(video_file), str(dir_path))
+                self._create_video_metadata(dir_path, video_file.name)
```

Comment:
To be sure, we are only writing the file's name in the metadata and ignoring all the relative paths.

Reply:
Yes, relative path is associated with dataset.